db/re/item_db.txt

```diff
@@ -528,7 +528,7 @@
 1170,Katzbalger,Katzbalger,5,20,,2000,175,,1,0,0x00004082,63,2,34,4,48,1,3,{ bonus bVit,5; bonus bDef,10; },{},{}
 1171,Zweihander_,Zweihander,5,20,,2200,200,,1,2,0x00004082,63,2,34,4,48,1,3,{ bonus bUnbreakableWeapon,1; },{},{}
 1172,Claymore_,Claymore,5,74000,,2500,180,,1,2,0x00004080,63,2,34,3,33,1,3,{},{},{}
-1173,Muramasa_C,Muramasa,5,1,,0,204,,1,0,0x00004082,63,2,34,4,1,0,3,{ bonus bCritical,30; bonus bAspdRate,8; },{},{}
+1173,Muramasa_C,Muramasa,5,1,,0,204,,1,0,0x00004082,63,2,34,4,1,0,3,{ bonus bCritical,30; bonus bAspdRate,8; bonus2 bAddEff,Eff_Curse,10; },{},{}
```

Comment:
This is iRO specific. If you look on divine-pride and switch server to kRO this effect is not mentioned in the item description.

Reply:
Confirmed, I'll change. :-)